pkg/kubectl/cmd/set/set_selector.go

```diff
@@ -109,6 +119,8 @@ func (o *SelectorOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args [
 	o.dryrun = cmdutil.GetDryRunFlag(cmd)
 	o.output = cmdutil.GetFlagString(cmd, "output")
 
+	o.PrintFlags.Complete(o.dryrun)
```

Comment:
Keep `Complete` and `ToPrinter` next to one another like in the previous set commands.